docs/geshi-doc.html

```diff
@@ -639,7 +640,7 @@ <h4 id="requirements">2.1.1 Requirements</h4><div class="nav"><a href="#getting-
 <h4 id="downloading-geshi">2.1.2 Downloading <abbr title="Generic Syntax Highlighter">GeSHi</abbr></h4><div class="nav"><a href="#requirements">Previous</a> | <a href="#getting-geshi">Top</a> | <a href="#extracting-geshi">Next</a></div>
 
 <p>There are several ways to get a copy of <abbr title="Generic Syntax Highlighter">GeSHi</abbr>. The first and easiest way of all is
-visiting <a href="http://qbnz.com/highlighter/downloads.php">http://qbnz.com/highlighter/downloads.php</a> to obtain the latest version.
+visiting <a href="https://sourceforge.net/projects/geshi/files">https://sourceforge.net/projects/geshi/files</a> to obtain the latest version.
```

Comment:
Not quite happy with either URL there. Better leave unchanged and open an issue for thorough fixing.

Background: gbnz.com is the actual homepage, SF.net is mostly abandoned and I'd prefer the GitHub link here; but Github doesn't have the old releases AND is lacking the signature files. Thus leave unchanged in this commit; has to be fixed another time.

Reply:
Understood. Reverted this part of the PR.